### DISCUSSION TITLE
excludeTables not working

### DISCUSSION THREAD

**Discussion by ChrisSantiago82** [spatie/laravel-backup#1442] [Q&A] [answered]
I have some tables that I want to exclude from the backup. Those are big tables but the data comes from an API so it's not worth using backup space for those tables.

I have excluded thos tables in the config file but they are still in the backup. I'm sure I'm missing something:

```
            'mysql' => [
                'dump' => [
                    'excludeTables' => [
                        'fashion_cloud_brands',
                        'fashion_cloud_products',
                        'fashion_cloud_product_images',
                        'fashion_cloud_product_image_files',
                    ]
                ],
            ],
            'databases' => [
                'mysql',
            ],
```

**Comment by ChrisSantiago82** [comment, marked as answer]
Sorry about this. The exludeTables must be added to the database config and not backup config

**Reply by PaolaRuby** [reply]
https://github.com/spatie/laravel-backup/issues/1368#issuecomment-959925953
